test/models/site_test.rb

```diff
@@ -92,4 +92,35 @@ def test_validate(data)
       assert_equal(expected, site.valid?)
     end
   end
+
+  sub_test_case "Custome style" do
+    test "with #base_hue" do
+      site = create(:site, base_hue: 300)
+
+      assert_valid_custom_css(site)
+    end
+
+    test "without #base_hue" do
+      site = create(:site, base_hue: nil)
+
+      assert_nil site.custom_hue_css_url
+    end
+
+    test "update #base_hue" do
+      site = create(:site, base_hue: nil)
+
+      assert_nil site.custom_hue_css_url
+      site.update!(base_hue: 200)
+      assert_valid_custom_css site
+      site.update!(base_hue: 300)
+      assert_valid_custom_css site
+      site.update!(base_hue: nil)
```

Comment:
We should assert that css file content is changed or not when `base_hue` is changed.